consensus/api/src/conversions.rs

```diff
@@ -266,7 +266,8 @@ mod conversion_tests {
     fn test_convert_active_mint_config() {
         let mut rng = Hc128Rng::from_seed([1u8; 32]);
         let (_mint_config_tx, signers) = create_mint_config_tx_and_signers(2.into(), &mut rng);
-        let signer_set = SignerSet::new(signers.iter().map(|s| s.public_key()).collect(), 1);
+        let signer_set =
+            SignerSet::new(signers.iter().map(|s| s.public_key()).collect(), vec![], 1);
```

Comment:
I think it is context dependent, but the guiding factor I personally use is to try and reduce the ways we can represent empty/"nothing" values. I think the only time I would add an Option, is if the empty value carries some special meaning I care about. In most cases I would prefer an empty string to indicate nothing, since often when I am passing a string around, I need it to contain something. If that is indeed a requirement, then I would have to check whether it is empty or not anyway so an Option wrapper does not get me much. Another way to put it is whether `None` or `""` gets passed around, I still don't have my string - unless an empty string really has some special meaning that is going to receive a treatment differently from `None. I believe that is fairly uncommon.